mappings/net/minecraft/world/border/WorldBorder.mapping

```diff
@@ -8,6 +8,7 @@ CLASS net/minecraft/class_2784 net/minecraft/world/border/WorldBorder
 	FIELD field_12736 area Lnet/minecraft/class_2784$class_2785;
 	FIELD field_12737 centerZ D
 	FIELD field_12738 centerX D
+	FIELD field_24122 PROPERTIES Lnet/minecraft/class_2784$class_5200;
```

Comment:
Who the heck named it? This is the default serialized form of world border, this name is horrible